det/llm/llm_openai.py

```diff
@@ -41,6 +41,10 @@ def __init__(self, model: str = "gpt-3.5-turbo", api_key: str = None):
         else:
             self.client = OpenAI()
         self.model = model
+        try:
```

Comment:
As per equivalent change in `det/embeddings/generator.py`

Reply:
change completed